package.json

```diff
@@ -14,8 +14,9 @@
   "repository": "git@github.com:rianadon/opensprinkler-card.git",
   "license": "Apache-2.0",
   "dependencies": {
+    "@formatjs/intl-utils": "^3.8.4",
     "@mdi/js": "^6.5.95",
-    "custom-card-helpers": "^1.8.0",
+    "custom-card-helpers": "1.8.0",
```

Comment:
Note: This is explicitly _removing_ the "compatible with" `^` qualifier, in favor of pinning the `custom-card-helpers` version at exactly 1.8.0. Version 1.9.0 ought to be compatible, _but_ took a major version bump of `home-assistant-js-websocket`, and we're exposed to the breaking changes due to data type incompatibility.